pkg/sinit/init_test.go

```diff
@@ -0,0 +1,46 @@
+package sinit
+
+import (
+	"context"
+	"testing"
+)
+
+func TestEmptyConfiguration(t *testing.T) {
+	ctx := context.Background()
+	cfg := LoggingConfig{}
+	InitializeLogging(ctx, cfg)
```

Comment:
shouldn't there be something like: `err := 	InitializeLogging(ctx, cfg)` logging here?